playkit/src/main/java/com/kaltura/playkit/player/PlayerController.java

```diff
@@ -942,7 +947,9 @@ private PlayerEngine.EventListener initEventListener() {
                 PKEvent event;
                 switch (eventType) {
                     case PLAYING:
-                        updateProgress();
+                        if (!PKMediaFormat.udp.equals(sourceConfig.mediaSource.getMediaFormat())) {
```

Comment:
For better readability add function isMulticast or isUDP.

Reply:
done